src/app/tests/suites/certification/Test_TC_EEVSEM_2_1.yaml

```diff
@@ -0,0 +1,219 @@
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Step 2: TH reads from the DUT the SupportedModes attribute."
+      PICS: EEVSEM.S.A0000
+      command: "readAttribute"
+      attribute: "SupportedModes"
+      response:
+          saveAs: supported_mode_dut
+          constraints:
+              type: list
+              minLength: 2
+
+    - label: "Step 3: TH reads from the DUT the CurrentMode attribute."
+      PICS: EEVSEM.S.A0001
+      command: "readAttribute"
+      attribute: "CurrentMode"
+      response:
+          saveAs: old_current_mode_dut
+          constraints:
+              type: int8u
+              minValue: 0
+              maxValue: 255
+
+    - label:
+          "Step 4: TH sends a ChangeToMode command to the DUT with NewMode set
+          to old_current_mode_dut"
+      PICS: EEVSEM.S.C00.Rsp
+      command: "ChangeToMode"
+      arguments:
+          values:
+              - name: "NewMode"
+                value: old_current_mode_dut
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+
+    - label:
+          "Step 5: Manually put the device in a state from which it will FAIL to
+          transition to PIXIT.EEVSEM.MODE_CHANGE_FAIL"
+      PICS:
+          " EEVSEM.S.M.CAN_TEST_MODE_FAILURE &&
+          EEVSEM.S.M.CAN_MANUALLY_CONTROLLED && PICS_SKIP_SAMPLE_APP "
+      verification: |
+          Manual operation required
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      arguments:
+          values:
+              - name: "message"
+                value: "Enter 'y' after success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "Step 6: TH reads from the DUT the CurrentMode attribute"
+      PICS: EEVSEM.S.A0001 && EEVSEM.S.M.CAN_MANUALLY_CONTROLLED
+      command: "readAttribute"
+      attribute: "CurrentMode"
+      response:
+          saveAs: Step6_current_mode_dut
+          constraints:
+              type: int8u
+              minValue: 0
+              maxValue: 255
+
+    #Skipping test-7 because SDK is not enabled with this failure response
+    - label:
+          "Step 7: TH sends a ChangeToMode command to the DUT with NewMode set
+          to PIXIT.EEVSEM.MODE_CHANGE_FAIL"
+      PICS:
+          EEVSEM.S.M.CAN_TEST_MODE_FAILURE && EEVSEM.S.C00.Rsp &&
+          PICS_SKIP_SAMPLE_APP
+      verification: |
+          Note : Please skip this step as SDK is not enabled with this failure response
+
+          ./chip-tool energyevsemode change-to-mode 2 1 1
+
+          Verify on TH(chip-tool) log, DUT responds contains a ChangeToModeResponse command with Status field is set to GenericFailure(0x02), InvalidInMode(0x03) , or in the MfgCodes (0x80 to 0xBF) range and StatusText field has a length between 1 and 64,  below is the sample log provided for the raspi platform:
```

Comment:
With the current YAML support, i'm not sure this step will EVER be possible to automate. So why go down the YAML route here at all if we know we're going to end up at python?